packages/expo-dev-launcher/android/src/debug/java/expo/modules/devlauncher/modules/DevLauncherInternalModule.kt

```diff
@@ -194,6 +192,17 @@ class DevLauncherInternalModule(reactContext: ReactApplicationContext?)
     promise.resolve(null)
   }
 
+  @ReactMethod
+  fun getMenuSettingsAsync(promise: Promise) {
+    promise.resolve(reactApplicationContext.getNativeModule(DevMenuSettings::class.java)?.serialize())
```

Comment:
this is a bit odd; could we just call into the `DevMenuSettings` module directly from JS? i'm not sure i understand the advantage of the indirection provided by this module.

Reply:
total brain fart - didn't realize I could just use DevMenuSettings as a native module - will update iOS to match!